types/heft-jest/index.d.ts

```diff
@@ -3,7 +3,7 @@
 // Definitions by: Pete Gonzalez <https://github.com/octogonz>
 //                 Ian Clanton-Thuon <https://github.com/iclanton>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.5
+// TypeScript Version: 3.8
```

Comment:
Heft currently ships with Jest 25. Seems like this change should wait until Heft upgrades its Jest dependency.

Reply:
Any idea of when that might be?

I don't think this'll be a problem so long as Heft can support TypeScript 3.8, which by the looks of it is can.